app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine-related errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor is not vaccinated."""
+    pass
```

Comment:
You should add a default message to your NotVaccinatedError. It could be something like 'The person is not vaccinated.'.